platform/cmd/scan_options.go

```diff
@@ -117,10 +117,8 @@ func ComputeFlags(cmd *cobra.Command, options *CliOptions) error {
 			&options.Image,
 			"image",
 			"",
-			"Defines an image to be used for analysis execution. \nSets --within-docker=true. Sets --linter to the one preinstalled within the image. \nAvailable images are: "+strings.Join(
-				product.AllImages,
-				", ",
-			)+".",
+			"Defines an image to be used for analysis execution. \nSets --within-docker=true. Sets --linter to the one preinstalled within the image. \nAvailable images are: jetbrains/qodana-jvm"+
+				product.JvmLinter.Image()+", "+product.DotNetLinter.Image()+", etc... Full list of images is available at https://hub.docker.com/u/jetbrains?search=qodana .",
```

Comment:
"etc" should end with a single period (not an ellipsis). But honestly I would just remove the examples and just direct the users to the webpage right away.